### DISCUSSION TITLE
开往状态页没有600以上的网站

### DISCUSSION THREAD

**Discussion by justin-jiajia** [travellings-link/travellings#1172] [寻求帮助] [answered]
如题。Id600以上是不是没有添加？
![3476A30B-D0FD-42E4-A92E-EA96C10DBF6B](https://user-images.githubusercontent.com/87896946/187107025-0872732a-2e2a-4ba5-b479-032140c10d15.jpeg)

**Comment by linlinzzo** [comment]
好像没有。我们添加网站的时候没有添加到【网站监控】的步骤。

**Reply by justin-jiajia** [reply]
所以...能添加一下吗？还是要等到ID到700以上才能添加600-699？

**Comment by linlinzzo** [comment, marked as answer]
我们暂时不会再更新这个监控了，等之后弄出自动化脚本。
